core/src/main/java/org/infinispan/transaction/xa/recovery/RecoveryAdminOperations.java

```diff
@@ -48,14 +48,13 @@ public String showInDoubtTransactions() {
          result.append("xid = [").append(i.getXid()).append("], ").append(SEPARATOR)
                .append("internalId = ").append(i.getInternalId()).append(SEPARATOR);
          result.append("status = [ ");
-         for (Integer status : i.getStatus()) {
-            if (status == Status.STATUS_PREPARED) {
-               result.append("_PREPARED_");
-            } else if (status == Status.STATUS_COMMITTED) {
-               result.append("_COMMITTED_");
-            } else if (status == Status.STATUS_ROLLEDBACK) {
-               result.append("_ROLLEDBACK_");
-            }
+         int status = i.getTxStatus();
```

Comment:
nitpick: we can use `org.infinispan.commons.tx.Util.transactionStatusToString()` to get the status as string.
if status != -1 :)